dev/io.openliberty.microprofile.config.2.0.internal_fat/fat/src/io/openliberty/microprofile/config/internal_fat/apps/brokenInjection/MissingConfigPropertyBean.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
```

Comment:
Note to reviewer(s): Git has muddled up a file creation + deletion for a file rename here.